README.md

```diff
@@ -6,6 +6,12 @@ The prototype kit provides a simple way to make interactive prototypes that look
 
 Read the [project principles](docs/principles.md).
 
+## How it differs from GOV.UK Prototype Kit
+
+This project is a fork of [GOV.UK Prototype Kit](https://github.com/alphagov/govuk_prototype_kit) so that we can benefit from the updates made to that project.
+
+The only difference is that it uses HMRC's [assets-frontend](https://github.com/hmrc/assets-frontend) instead of GOV.UK Elements as that's the library our frontends use in production.
+
 ## Security
 
 If you publish your prototypes online, they **must** be protected by a [username and password](docs/guides/publishing-on-heroku.md). This is to prevent members of the public finding prototypes and thinking they are real services.
```

Comment:
I'm not sure what you mean. What is the standard? Get what from updating the fork?
The HMRC prototype kit is a fork of the GOV.UK prototype kit but without the static assets from GOV.UK elements that the GOV.UK prototype kit has. 
HMRC prototype kit contains assets frontend to match the assets we have in production. In turn assets frontend has the GOV.UK dependencies which need updating. Updating these dependencies inside of assets frontend is a body of work that we are looking at next. 
Please read the blog posts around Assets frontend version 3 to understand the dependency relationships assets frontend has